akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -309,6 +317,9 @@ object TestSubscriber {
       self
     }
 
+    def expectMsgPF[T](f: PartialFunction[AnyRef, T]): T =
+      f(probe.expectMsgType[AnyRef])
```

Comment:
shouldn't this be `expectNextPF` and handle the unwrapping, so that the PartialFunction is written in terms of elements?